### ISSUE TITLE
Option to create create categories for transcript in setting for context & email organization

### ISSUE THREAD

**Issue by ammaraslam** [chatwoot/chatwoot#4198]
**Is your feature request related to a problem? Please describe.**
So whenever a transcript is being sent to some email, it's just agent's and users messages but we want to know what that chat is all about without going through complete transcript, it would save my team a lot of time considering the volume of chat.

**Describe the solution you'd like**
Ability to select categories for each inbox after one selects send transcript (Using "Angry Customer" category for "Angry customers" or something else) & one shouldn't have to make categories every time to send transcripts, they'll be created by admin only & but can be selected by anyone regardless of roles.

**Additional context**
After the transcript is send with category selected, it should appear in email as under,

Name : "Name provided on chat"
Category : "Selected Category"
Phone number : "If available"
Email "If available"

Transcript : .... ..... ..... ......

TLDR ; Select category while sending transcript to better organize and increase productivity of team.

**Comment by sojan-official** [comment]
@ammaraslam  could you give more information about how you are using the transcript functionality?  are you using it for escalation or working over the transcripts in an external system? 

Also would the ability to attach a summary note to the transcripts solve the issue?

**Comment by ammaraslam** [comment]
@sojan-official every chats transcript is sent to our internal team, what I'm tryna achieve to quick organization of emails and ability to know the issue of visitors without reading the full transcript -- suppose visitor is coming for the pricing issue, and there's category in which pricing issue is selected, my team will instantly send that email to finance department -- Can't say much about the note, wouldn't be able to organize like categories.

**Comment by sojan-official** [comment]
@ammaraslam will this workflow work in your case? 

- the internal team also have access to the chatwoot account. Rather than sending the transcript, the agents can add a private note with the summary and then assign the conversation to the appropriate team. 
- You could also additionally tag the conversations with labels.

**Comment by ammaraslam** [comment]
It would've, but considering all company contacts are being shown to
agents, which we really don't want to, due to major privacy reasons.

How about sending added label with transcript.

Something like this 👇

Label : Billing Issue

Transcript:

..........
.........

**Comment by sojan-official** [comment]
@ammaraslam  Thanks a lot for all the feedback. We will keep it in our backlog and prioritise accordingly. 

ref: #4216

**Comment by stonecharioteer** [comment]
This issue is being closed due to lack of activity for over a year. [For more details, visit this discussion.](https://github.com/orgs/chatwoot/discussions/12810)
